doc_user/UserManual.adoc

```diff
@@ -154,19 +154,19 @@ The following fields are available to select or narrow the resulting matches:
 * **Date. “**DATE From” and “DATE To” allows for the selection of all visits performed during the specified period
 * **Age. “**Age From” and “Age To” allows for the selection of the patients by age
 * *Sex.* Allows for the selection of all the patients by sex: All / Male / Female
-* *Type of patient.* Allows for the selection of all the visits by type of attendance: All / New / Re-Attendance
+* *Type of patient.* Allows for the selection of all the visits by type of attendance: All / New / Readmittance
```

Comment:
Here the term "Re-Attendance" is the one commonly used to refer to OPD cases: it indicates a subsequent (visit) concerning the same medical case (same disease) or a planned followup.

Reply:
Readmittance is a more appropriate English word for "re-attendance" (which BTW is only correct without the "hyphen") and I have been changing it in the resource bundle files.

Reply:
please consider this term as not-questionable at the moment: we'll use Re-Attendance (with hyphen) when we deal with OPDs.
Readmittance could be used for subsequent Admissions, but they are not yet labelled as such in the application (yet).

Reply:
Done